### ISSUE TITLE
Webkit removed from deprecated Qt4

### ISSUE THREAD

**Issue by 3vi1** [Xanthos-Eve/vintel#90]
Newer builds in Debian, Ubuntu, etc. have removed the webkit components (http://metadata.ftp-master.debian.org/changelogs//main/p/python-qt4/python-qt4_4.11.4+dfsg-2_changelog) for support reasons in the deprecated Qt4 libs (https://lists.debian.org/debian-devel-announce/2015/05/msg00001.html).

Are there plans to move Qt5?  The current version can't be run on newer Linux distros as it will now throw "ImportError: No module named QtWebKit" when importing from PyQt4.QtWebKit (as tested on Ubuntu 16.10).

**Comment by 3vi1** [comment]
I took a stab at it earlier and got it running with Qt5.  It runs, but I need to do actual testing to make sure I didn't break/miss anything.

Are there any issues with using Qt5 on the other platforms?  If not, I could send you a pull request after I do QA and push it to my fork.

**Comment by Xanthos-Eve** [comment]
Hi @3vi1,

@Xaroth recently led the charge on some changes to ease the transition to python 3 by cutting over to using modules Requests and Six. I like your proposal but I would not want those changes going into master since that is the release branch. Instead I would like to use the develop branch for this. Please go ahead and submit the PR against develop. This way if there are bug fixes to be made in master, we can do that and push it out. We can be testing and preparing develop for its eventual merge into master in parallel to regular bug fixing activities.

**Comment by 3vi1** [comment]
Sounds good.  I didn't get a chance to play around with testing last night, but I should get to it by this weekend and if I haven't added new bugs <g> I'll look at staging things for the develop branch.  It's 400+ changed lines, mostly due to the new-style signalling/slot format.

**Comment by 3vi1** [comment]
https://github.com/Xanthos-Eve/vintel/pull/92

**Comment by 3vi1** [comment]
Okay, I've submitted a new pull based on the current develop branch:  https://github.com/Xanthos-Eve/vintel/pull/102

**Comment by Xanthos-Eve** [comment]
Ok its in - time for me to install the newer sdks and give it a go!
